src/game_interpreter_battle.cpp

```diff
@@ -605,12 +606,42 @@ bool Game_Interpreter_Battle::CommandManiacControlAtbGauge(lcf::rpg::EventComman
 	return true;
 }
 
-bool Game_Interpreter_Battle::CommandManiacChangeBattleCommandEx(lcf::rpg::EventCommand const&) {
+bool Game_Interpreter_Battle::CommandManiacChangeBattleCommandEx(lcf::rpg::EventCommand const& com) {
 	if (!Player::IsPatchManiac()) {
 		return true;
 	}
 
-	Output::Warning("Maniac Patch: Command ChangeBattleCommandEx not supported");
+	// 1 row removed
+	bool actorCommandFlags = com.parameters[0];
```

Comment:
Style:

Variables are usually ``snake_case`` and functions ``UpperPascalCase``.

When it is a lambda function we are a bit indifferent because syntax wise its a mix of a variable and a function :sweat_smile: 

``{`` is usually on the same line as the ``condition``. But this is a bit mixed in our code...

Reply:
Yeah, I looked a around a bit, noticed it was used in one place, copied it over and started using it without fully understanding the context of how it's used in different places. My bad 😅 I've fixed it now in all the places I could find (not just this specific one you called out).